tests/typings/sync/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "outDir": "dist",
     "noImplicitAny": true,
+    "esModuleInterop": true,
```

Comment:
let's remove this, I don't see a need for it.
```suggestion
```

Reply:
This was the result of Typescript requiring `@types/allure-js-commons`.